docs/developing.md

```diff
@@ -367,6 +367,9 @@ Once you have finished running the above command, Websurfx should now be served
 
 > **Note**
 > When you build the project first time with the above command it will require the app to compile every dependency in the project which will then be cached on your system. So when you compile the app next time it will only compile for the new changes.
+> +
++ - `cargo run`: Starts the app and serves the project on http://127.0.0.1:8080.
+
```

Comment:
```suggestion
>
- `cargo run`: Starts the app and serves the project on http://127.0.0.1:8080.

```

Ok, I would suggest one last minor suggestion, I would suggest removing the plus signs as well and after that we will merge this PR in :slightly_smiling_face: .

Reply:
done!